src/theme/styles/global.ts

```diff
@@ -19,29 +19,13 @@ const selection = css`
   /* End CSS for selecting text */
 `
 
-const scrollbars = css`
```

Comment:
Scrollbars were being applied site wide. Revert back to browser defaults. Apply only in cases where we want to deviate from this (e.g. inner scrolling tables).